README.md

```diff
@@ -12,7 +12,7 @@ This framework allows applications to log schematized events, rather than textua
 ETW analysis tools can reliably identify fields within your events, and treat them as
 strongly-typed data, rather than text strings.
 
-To use [tracing](https://tracing.rs) with ETW, see [tracing-etw](./win_etw_tracing/).
+To use [tracing](https://tracing.rs) with ETW, see [tracing-etw](https://github.com/microsoft/tracing-etw).
```

Comment:
Is https://github.com/microsoft/rust_win_etw/blob/main/win_etw_tracing/Cargo.toml#L6 also a crate to use tracing with ETW? What is the difference between both?

Reply:
Claiming to link to the tracing-etw crate, and then linking to something different, is not good. Especially when that other link doesn't work in the first place.

This crate's own readme says to use tracing-etw, not its own tracing-interop layer. Having read through it thoroughly, I have suspicions why this may be, but they are not relevant to getting the tracing-etw link fixed.

Reply:
> This crate's own readme says to use tracing-etw, not its own tracing-interop layer

Can you share the link which says so?

Are you saying https://github.com/microsoft/rust_win_etw/blob/main/win_etw_tracing/Cargo.toml#L6 should not be used? Or both are valid?

Reply:
> To use [tracing](https://tracing.rs) with ETW, see [tracing-etw]

It does not say to use `win_etw_tracing`, it says to use `tracing-etw`. You will have to ask the author why; I do not want to speak on his behalf.

Reply:
I'd let the maintainers of this repo clarify which one is recommended or there is no such recommendation.